assets/js/script.js

```diff
@@ -129,10 +129,10 @@ const handleChoice = async (choiceIndex) => {
   let result = selectedStory[currentSceneKey].choices[choiceIndex].result;
   document.getElementById("storyboard").innerText = `${action}. ${result}`;
 
-  await new Promise(resolve => setTimeout(resolve, 6000));
+  //await new Promise(resolve => setTimeout(resolve, 6000));
```

Comment:
Can you uncomment these so we dont add broken code to main please

Reply:
done